src/pages/view-test-result/cat-a-mod1/components/business-details-card/__tests__/business-details-card.spec.ts

```diff
@@ -0,0 +1,139 @@
+
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { IonicModule, Config, Platform } from 'ionic-angular';
+import { ConfigMock, PlatformMock } from 'ionic-mocks';
+import { MockComponent } from 'ng-mocks';
+import { DataRowComponent } from '../../../../../../components/common/data-row/data-row';
+import { DataRowCustomComponent } from '../../../../../../components/common/data-row-custom/data-row-custom';
+import { BusinessDetailsCardComponent } from '../business-details-card';
+// todo: Change to CatAMod1UniqueTypes when schema changes are ready
```

Comment:
We labeled all the todos with `PREP-AMOD1` just so that all A Mod1 todo pops up in a common search

Reply:
Thanks Szabi, I'll correct this now.